### PR TITLE
LibJS: Remove most includes of GlobalObject.h

### PR DESCRIPTION
**Pull request by Hendiadyoin1**
Note: Somethings up with the function pointer declarations and I am not sure which is more correct

### CODE REVIEW

File: Userland/Libraries/LibJS/Runtime/ValueInvoke.h [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#include <LibJS/Heap/MarkedVector.h>
```
Comment (ADKaster) [inline Userland/Libraries/LibJS/Runtime/ValueInvoke.h:1]: Needs license and pragma once

Comment (Hendiadyoin1) [reply Userland/Libraries/LibJS/Runtime/ValueInvoke.h:1]: a shoot

Comment (trflynn89) [reply Userland/Libraries/LibJS/Runtime/ValueInvoke.h:1]: Any reason not to just put this in ValueInlines.h?

Comment (Hendiadyoin1) [reply Userland/Libraries/LibJS/Runtime/ValueInvoke.h:10]: 1: I added a generic license header, as tracking ownership of that function felt a bit tedious
2: This header is a bit more heave than the other inlines, so I feel like keeping it separate is nicer

Comment (trflynn89) [reply Userland/Libraries/LibJS/Runtime/ValueInvoke.h:10]: > This header is a bit more heave than the other inlines, so I feel like keeping it separate is nicer

Eh, disagreed, it's now one more special header we need to remember. Seems nicer to me to just have this in ValueInlines.

**Pull request closed without merging**